.travis.yml

```diff
@@ -44,7 +44,7 @@ matrix:
         - TESTMODE=fast
         - COVERAGE=
         - NPY_RELAXED_STRIDES_CHECKING=1
-        - NUMPYSPEC="git+git://github.com/numpy/numpy.git@v1.9.1"
+        - NUMPYSPEC="--upgrade git+git://github.com/numpy/numpy.git@v1.9.1"
```

Comment:
TBH I've no idea --- the ways of pip are a mystery to me :-). 

All I'm guessing is that pinning a version to build is OK if augmented by either a nosetest `assert that strides are relaxed` or maybe there is a lighter touch solution to convert a python assertion into a bash level return code.